tests/phpunit/tests/user/retrievePassword.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Test retrieve_password, in wp-includes/user.php
+ *
+ * @group user
+ */
```

Comment:
```suggestion
/**
 * Test retrieve_password(), in wp-includes/user.php.
 *
 * @group user
 *
 * @covers ::retrieve_password
 */
```
A couple of formatting updates and addition of `@covers` annotation.

Reply:
Done!